scr/main/java/bittercode/constant/ResponseCode.java

```diff
@@ -0,0 +1,42 @@
+package com.bittercode.constant;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum ResponseCode {
+
+    SUCCESS(200, "OK"),
+    FAILURE(422, "Unprocessible Entity, Failed to Process"),
+    PAGE_NOT_FOUND(404, "The Page You are Searching For is Not available"),
+    ACCESS_DENIED(403, "Please Login First to continue"),
+    BAD_REQUEST(400, "Bad Request, Please Try Again"),
+    INTERNAL_SERVER_ERROR(500, "Internal Server Error, Try Again!!"),
+    DATABASE_CONNECTION_FAILURE(406,
+            "Unable to Connect to DB, Please Check your db credentials in application.properties"),
+    METHOD_NOT_ALLOWED(405, "Requested HTTP method is not supported by this URL"),
+
+    ;
+
+    private final String message;
+    private final int ;
+
+    ResponseCode(int code, String message) {
+        this.message = message;
+        this.code = code;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public int getCode() {
+        return code;
+    }
+
+    // Reverse lookup methods
+    public static Optional<ResponseCode> getMessageByStatusCode(int statusCode) {
+        return Arrays.stream(ResponseCode.values())
+                .filter(error -> error.getCode() == statusCode)
+                .findFirt<>;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/2#issuecomment-2419037001>#be98fd</a><span> - 10/17/2024, 09:42 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> There seems to be an issue with the method name on line 40. It might be worth reviewing this part of the code.
</div>
<div id="fix">
<b>Fix:</b> Correct the method name from 'findFirt<>' to 'findFirst()'
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -40 +40 @@
 -                .findFirt<>;
 +                .findFirst();
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged